docs/whats-new/dotnet-docs-2022-01-01.md

```diff
@@ -0,0 +1,75 @@
+- [CA1843: Do not use 'WaitAll' with a single task](../fundamentals/code-analysis/quality-rules/ca1843.md)
+- [CA1848: Use the LoggerMessage delegates](../fundamentals/code-analysis/quality-rules/ca1848.md)
+- [CA2017: Parameter count mismatch](../fundamentals/code-analysis/quality-rules/ca2017.md)
+- [CA2253: Named placeholders should not be numeric values](../fundamentals/code-analysis/quality-rules/ca2253.md)
+- [CA2254: Template should be a static expression](../fundamentals/code-analysis/quality-rules/ca2254.md)
+- [CA2256: All members declared in parent interfaces must have an implementation in a DynamicInterfaceCastableImplementation-attributed interface](../fundamentals/code-analysis/quality-rules/ca2256.md)
+- [CA2257: Members defined on an interface with the 'DynamicInterfaceCastableImplementationAttribute' should be 'static'](../fundamentals/code-analysis/quality-rules/ca2257.md)
+- [CA2258: Providing a 'DynamicInterfaceCastableImplementation' interface in Visual Basic is unsupported](../fundamentals/code-analysis/quality-rules/ca2258.md)
+- [Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv and libuv.dll removed](../core/compatibility/aspnet-core/7.0/libuv-transport-dll-removed.md)
+- [SYSLIB0037: AssemblyName members HashAlgorithm, ProcessorArchitecture, and VersionCompatibility are obsolete](../fundamentals/syslib-diagnostics/syslib0037.md)
+- [Some APIs throw ArgumentNullException (.NET 7)](../core/compatibility/windows-forms/7.0/apis-throw-argumentnullexception.md)
+- [API obsoletions with non-default diagnostic IDs (.NET 7)](../core/compatibility/core-libraries/7.0/obsolete-apis-with-custom-diagnostics.md)
+- [SYSLIB0036: Regex.CompileToAssembly is obsolete](../fundamentals/syslib-diagnostics/syslib0036.md)
+- [Deserialize Version type with leading or trailing whitespace](../core/compatibility/serialization/7.0/deserialize-version-with-whitespace.md)
+- [Set timestamp on read-only file on Windows](../core/compatibility/core-libraries/6.0/set-timestamp-readonly-file.md)
+- [.NET implementations](../fundamentals/implementations.md)
+- [Ecma standards](../fundamentals/standards.md)
+- [Breaking changes in .NET 7](../core/compatibility/7.0.md)
+- [Validate CompressionLevel for BrotliStream](../core/compatibility/core-libraries/7.0/compressionlevel-validation.md)
+- [Default serialization format for TimeSpan](../core/compatibility/serialization/6.0/timespan-serialization-format.md)
+- [IL3003: 'RequiresAssemblyFilesAttribute' annotations must match across all interface implementations or overrides.](../fundamentals/code-analysis/quality-rules/il3003.md)
+
+### Updated articles
+
+- [Upgrade a Windows Forms App to .NET 6 with the .NET Upgrade Assistant](../core/porting/upgrade-assistant-winforms-framework.md) - Upgrade assistant updates
+- [Upgrade a WPF App to .NET 6 with the .NET Upgrade Assistant](../core/porting/upgrade-assistant-wpf-framework.md) - Upgrade assistant updates
+
+## Community contributors
```

Comment:
Add an announcement of updating the source for the C# specification:

```suggestion
## C# language

### Updated articles

- [ECMA C# (draft) standard](csharp/language-reference/language-specification/readme) - Publish the C# draft standard instead of the Microsoft C# draft specification.

## Community contributors
```